### PR TITLE
Added validation and error handling. Standardized response object

### PR DESCRIPTION
**Pull request by sreemukha**

### CODE REVIEW

File: tsds-grafana-handler/tsds_grafana_handler.py [hunk 1]
```diff
@@ -200,12 +200,11 @@ def bucket_size(size):
...
-                        alias = positions[indexes.pop(0)]
-                        swap  = aliases[alias]
-                        del aliases[alias]
-                        alias = alias.replace('$quantify', replace_val)
-                        aliases[alias] = swap
+                alias = positions[indexes.pop(0)]
+                swap  = aliases[alias]
+                del aliases[alias]
+                alias = alias.replace('$quantify', replace_val)
+                aliases[alias] = swap
```
Comment (daldoyle) [inline tsds-grafana-handler/tsds_grafana_handler.py]: I think something actually got lost here, the if len(indexes) > 0 part. Be careful when doing cp that you are running from the latest version :)

If you can get this added, I think the rest looks good.

**Review by daldoyle — CHANGES_REQUESTED** [review]
See comments

File: tsds-grafana-handler/tsds_grafana_handler.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#/usr/bin/python
```
Comment (daldoyle) [inline tsds-grafana-handler/tsds_grafana_handler.py]: Why was this changed? The shebang was the correct syntax. It is not a comment :)

**Review by daldoyle — CHANGES_REQUESTED** [review]
See comments

File: tsds-grafana-handler/tsds_grafana_handler.py (hunk 2 above)
Comment (sreemukha) [reply tsds-grafana-handler/tsds_grafana_handler.py]: Sorry. I accidentally deleted it. I fixed it.
